imap_processing/swe/tests/test_decom.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+
+import pytest
+
+from imap_processing import decom
+
+
+@pytest.fixture(scope="session")
+def decom_test_data():
+    """Read test data from file"""
+    packet_file = Path(
+        "imap_processing/swe/tests/science_block_20221116_163611Z_idle.bin"
+    )
+    xtce_document = Path("imap_processing/swe/swe_packet_definition.xml")
+    data_packet_list = decom.decom_packets(packet_file, xtce_document)
+    return data_packet_list
```

Comment:
If you are going to move the packet definitions into a single area like `imap_processing/packet_definitions/swe_packet_definition`, I'd suggest making a `swe_decom.py` file that automatically points to that and just takes in the binary file as input. That way the test doesn't hardcode in the path to the xtce document, but that lives with SWE code.

```suggestion
    data_packet_list = swe_decom.decom_packets(packet_file)
    return data_packet_list
```

Reply:
I had that initially but I removed it because it was it wasn't doing much besides returning the list. But I can add it back in.

Reply:
Ahh, sorry. I mean to keep this new `decom.py` file you made here and add another abstraction layer so other groups can copy your template for their instrument. (Before you had it all in SWE, so now I'm suggesting to put a little bit in each area)

Reply:
Sorry for the misunderstanding. In my last commit, I had that change but took it out. Adding it back it with minor improvement based on feedback.

Reply:
Nice, Tenzin! I like this organization, looks good and my approval still stands :)